### PR TITLE
Creation Comment entity complete ticket OT-147-76

### PR DESCRIPTION
**Pull request by walterio123**
**Summary**
- Controller creation for Comment.
- Creation of the route to create Comment.
- Creation method service for creation of CommentEntity.
- Creation Dto fot CommentEntity.
- Creation Mapper for CommentEntity.
- Add path to security only accessible for User role.

### CODE REVIEW

File: src/main/java/com/alkemy/ong/controller/CommentController.java [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    //Create news entity only by User.
+    @PostMapping()
+    public ResponseEntity<CommentDto> create(@RequestBody @Valid CommentDto dto) {
+        try {
+            return ResponseEntity.ok(commentService.save(dto));
+        } catch (Exception e) {
+            return new ResponseEntity<>(HttpStatus.UNPROCESSABLE_ENTITY);
+        }
+    }
+
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/CommentController.java:29]: There is no need for empty lines

File: src/main/java/com/alkemy/ong/controller/CommentController.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+import javax.validation.Valid;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/comments")
+public class CommentController {
+    @Autowired
+    private  CommentService commentService;
+    //Create news entity only by User.
```
Comment (anavduo) [inline src/main/java/com/alkemy/ong/controller/CommentController.java:20]: Shouldn't it say Comment Entity?

Comment (walterio123) [reply src/main/java/com/alkemy/ong/controller/CommentController.java:20]: You're right, I've fixed it.

File: src/main/java/com/alkemy/ong/controller/CommentController.java (hunk 1 above)
Comment (walterio123) [reply src/main/java/com/alkemy/ong/controller/CommentController.java:29]: You're right, I've fixed it.

**Comment by pablofrankenn** [comment]
It's ok for me.

**Comment by anavduo** [comment]
Nice job!

**Pull request merged**